### PR TITLE
Run integration tests in CI

### PR DESCRIPTION
**Pull request by allysonjp715**
Requires changing the test to an add rather than edit test, because the stack that's brought up with docker-compose doesn't have any existing rows in the table. Will clean up this discrepancy between local / CI in a future PR.  

Will also remove the docker.yaml file once we add an integration test for /sources.

### CODE REVIEW

File: verification/curator-service/api/src/util/validate-env.ts [hunk 1]
```diff
@@ -26,10 +26,12 @@ export default function validateEnv(): Readonly<{
         PORT: port({ default: 3001 }),
         GOOGLE_OAUTH_CLIENT_ID: str({
             desc: 'OAuth client ID from the Google developer console',
+            default: 'replace to enable auth',
```
Comment (attwad) [inline verification/curator-service/api/src/util/validate-env.ts:29]: no please outside of dev we have to make sure we put in the right values here, same below

File: verification/curator-service/ui/package.json [hunk 2]
```diff
@@ -26,11 +26,14 @@
...
     "start": "PORT=3002 react-scripts start",
+    "start-full-stack": "./../../../dev/run_stack.sh",
     "build": "react-scripts build",
     "test": "react-scripts test",
     "test-silent": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
-    "cypress": "cypress open"
+    "cypress-open": "cypress open",
+    "cypress-run": "cypress run",
+    "cypress-ci-test": "WAIT_ON_TIMEOUT=600000 start-server-and-test start-full-stack http://localhost:3002 cypress-run"
```
Comment (attwad) [inline verification/curator-service/ui/package.json:36]: building and starting the stack should be a github actions step, cf how the current docker workflow work:
https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/.github/workflows/docker.yml#L18

once you've done that then you can npm run cypress-ci-test that points to localhost:3002, no need for start-server-and-test and start-full-stack scripts anymore

File: verification/curator-service/api/src/util/validate-env.ts (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/api/src/util/validate-env.ts]: Done

File: verification/curator-service/ui/package.json (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/package.json]: Ahh thanks! Done.

File: .github/workflows/integration-tests.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```
Comment (attwad) [inline .github/workflows/integration-tests.yml:1]: remove stale comment or comment about cypress/usage of this workflow?

File: dev/test_all.sh [hunk 4]
```diff
@@ -2,4 +2,5 @@
 set -e
 npm --prefix=`dirname "$0"`/../verification/curator-service/api/ run-script test-silent
 npm --prefix=`dirname "$0"`/../data-serving/data-service/ run-script test-silent
-npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script test-silent
\ No newline at end of file
+npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script test-silent
+npm --prefix=`dirname "$0"`/../verification/curator-service/ui/ run-script cypress-ci-test
```
Comment (attwad) [inline dev/test_all.sh:6]: s/cypress-ci-test/cypress-run ?

**Comment by allysonjp715** [comment]
[Integration testing strategy](https://trello.com/c/dCCp9gpE/134-integration-testing-strategy)

File: .github/workflows/integration-tests.yml (hunk 3 above)
Comment (allysonjp715) [reply .github/workflows/integration-tests.yml]: Removed

File: dev/test_all.sh (hunk 4 above)
Comment (allysonjp715) [reply dev/test_all.sh]: Good catch. Also added comment that local stack is necessary for that command.

**Pull request merged**